### PR TITLE
Add a prefix to the grouping id, making matching more specific

### LINKED ISSUE #31: Environment chooser too liberal in what it shows
When you have two environments named "Pre-Production" and "Production", the environment page shows "Production" environment (as well as the expected "Pre-Production") when you select "Pre-Production".

This is because [the code](https://github.com/davidroberts63/OctoPygmy/blob/master/src/commonpygmy.js#L33) is doing an indexOf, which seems is due to the fact the code can handle it being a string or an array.

### PR DESCRIPTION
**Pull request by matt-richardson**
Fixes #31

Also moved duplicated function into common module

### CODE REVIEW

**Comment by davidroberts63** [comment]
Ah yes. I had that fixed for the environments in 2.0 but didn't push that to 3.0. Good catch. I'll publish this one out with your other work tonight or tomorrow morning.

File: src/dashboard-collapser.js [hunk 1]
```diff
@@ -2,10 +2,6 @@ var dashboardCollapser = {
 	chooserId: "project-chooser",
 	projectGroupIds: [],
 
-	groupingId: function(groupName) {
```
Comment (davidroberts63) [inline src/dashboard-collapser.js]: You removed the function here but it's being called on line 26? Did you mean to point it to commonpygmy.groupId?

Comment (matt-richardson) [inline src/dashboard-collapser.js]: Good catch - completely missed that one.
Also missed in testing - I didn't test against 2.x.

Probably worth trying to up the test coverage around the collapsers.

Comment (matt-richardson) [inline src/dashboard-collapser.js]: Fixed.

**Pull request merged**